draft-ietf-httpbis-digest-headers.md

```diff
@@ -271,7 +271,9 @@ representation metadata, payload transformations and HTTP methods on Digest is
 provided in {{examples-unsolicited}} and {{examples-solicited}}.
 
 A `Digest` field MAY contain multiple representation-data-digest values. This
-could be useful for responses expected to reside in caches shared by users with
+could be useful
+to transition from weak algorithms or
+for responses expected to reside in caches shared by users with
 different browsers, for example.
```

Comment:
If we're going to expand this we can probably write this better. The first thing that spring into my head is something like

A `Digest` field MAY contain multiple representation-data-digest values. For example, a server may provide representation-data-digest values using different algorithms, allowing it to support a population of clients with different evolving capabilities.

Reply:
and optionally tack on

"This is particularly useful in support of transitioning away from weaker algorithms should the need arise; see {{algorithm-agiity}}"

Reply:
Done